packages/builder/src/components/design/settings/controls/DraggableList/DraggableList.svelte

```diff
@@ -100,9 +100,6 @@
       on:click={() => {
         get(store).actions.select(draggableItem.id)
       }}
-      on:mousedown={() => {
-        get(store).actions.select()
-      }}
```

Comment:
Not needed - this was just closing the previous popover on mouse down which isn't how we usually handle popovers so I'm removing this to keep things consistent with just clicks.